__tests__/app.test.js

```diff
@@ -105,6 +105,58 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: votes updated with a positive value", () => {
+    const updatedVotes = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updatedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.votes).toBeGreaterThan(0);
+        expect(body.article.article_id).toBe(1);
+      });
+  });
```

Comment:
Nice, but we should be asserting the entire article object here instead of just two properties